.circleci/config.yml

```diff
@@ -2172,7 +2176,13 @@ workflows:
   test-intg:
     jobs:
       - test-intg-downstream
-      - test-intg-master
+      - test-intg-master:
+          name: test-intg-master
+      - test-intg-master:
+          name: test-intg-master-upstream-only
+          make_target: "test-intg-upstream"
+          context: storage-unit-tests
+          filters: *any-upstream
```

Comment:
One of the new tests needs the AWS credentials that are only upstream. I can just skip it when not upstream like Ryan suggested rather than adding another target.